### ISSUE TITLE
 🐞 Copy All Content does not work

### ISSUE THREAD

**Issue by lensbreak** [elementor/elementor#24879]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

If you try to copy all content and paste it on a new page, it only pastes the first container, not the whole page content. Happens on plain vanilla Elementor Pro + Hello Theme website, so it is not a plugin conflict but a bug.

### Steps to reproduce

1. Edit any page with some containers and content on it
2. Right-click on the dashed box at the end of the page
3. Select "Copy All Content" from the context menu
4. Open any other (empty) page within the same domain/installation
5. Right-click on the page and select "Paste"
6. Only the first container of the copied page appears. 

### Expected behavior

After pasting, all the containers of the copied page should appear. This worked very well in the past. Did not test it with cross-site copy/paste all content.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MySQL Community Server - (GPL) v5.7.41
	PHP Version: 8.1.26
	PHP Memory Limit: 4096M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 80M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.2
	Site URL: https://
	Home URL: https://
	WP Multisite: No
	Max Upload Size: 80 MB
	Memory limit: 80M
	Max Memory limit: 4096M
	Permalink Structure: /%postname%/
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.0.0
	Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: de_CH
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.18.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.18.3
		Author: Elementor.com

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Active by default
	Build with AI: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Optimize Image Loading: Inactive
	Pages Panel: Inactive by default
	Global Style Guide: Active
	Page Transitions: Inactive
	Notes: Inactive
	Form Submissions: Active by default
	Scroll Snap: Inactive
	Menu: Active
	Taxonomy Filter: Inactive


== Log ==
	


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by KingGrizzly** [comment]
@lensbreak Have you tried this in a different browser? Also, does the page you are copying have a lot of content? I've encountered this issue if I try to copy a large quantity of containers which has required me to multi-select containers, and paste them over in batches. For example, copy and paste about half of the containers and then the rest.

Or sometimes I seem to need a new browser session.

Just curious if browser or quantity of containers being pasted makes any difference for you.

I just tested "Copy All Content" paste and it worked for me. On cross-site paste the gaps get messed up, but it seems to work on the same site.

**Comment by SteidleSolutions** [comment]
Same problem here. It only copies the first section and skips everything else.

**Comment by cajubaena** [comment]
Same problem here

**Comment by raj1eev2** [comment]
I am also facing the same issue while pasting container to another page.  I get error message 'Something went wrong'

**Comment by KingGrizzly** [comment]
Yes, I get the same thing now when using "Copy All Content" on a single site, only the first container pastes when copying from one page to another.

There are also fundamental cross site pasting issues existing since October of last year: https://github.com/elementor/elementor/issues/24111

**Comment by 319heads** [comment]
Same problem here. So frustrating that basic essential features like this, that drastically affect workflow speed, continue to be broken in recent versions of Elementor.

**Comment by KingGrizzly** [comment]
Noticing that copy and paste styles is now failing as well. So that makes: cross site copy/paste, cross site export and import, as well as copy all content all broken.

**Comment by peixotorms** [comment]
still happening

**Comment by KingGrizzly** [comment]
Elementor support was able to confirm cross-site copy and paste bugs after watching this and trying my demo environments. 

https://www.loom.com/share/adda9c50442b4135b2ab10dec759d42b?sid=852315b8-0483-4716-b76a-b99487d6110e

So this means we have in-site and cross-site copy/paste bugs as well as cross-site export/import bugs. Core features not working. All hands on deck type of problems, imo.

**Comment by captainplod** [comment]
Has anyone found a solution for this issue?
